GVFS/GVFS/CommandLine/DehydrateVerb.cs

```diff
@@ -114,12 +123,22 @@ protected override void Execute(GVFSEnlistment enlistment)
                     return;

```

Comment:
It looks like the above `"WARNING: THIS IS AN EXPERIMENTAL FEATURE` message needs to be updated (or skipped) when using the `Folders` option.